CHANGELOG.md

```diff
@@ -24,8 +24,10 @@
 * Handle duplicate record fields in GHC 9.8 instead of crashing
 
 ## 0.4.4.0 -- 2023-11-13
+
```

Comment:
This is the first release following drop of support from hie compat several months before this was released, so adding it here retrospectively.